daemon/src/lib.rs

```diff
@@ -350,6 +351,11 @@ where
             })
             .await?
     }
+
+    pub async fn sync_wallet(&self) -> Result<()> {
+        self.wallet_actor.send(wallet::Sync).await?;
```

Comment:
My understanding is, that this is a blocking `send` that we await so the UI gets feedback if the sync is done.
Not sure how the routes will behave if a user triggers some other actions while waiting for the wallet to sync, but I don't think it matters much.
I was thinking if it would make sense to make this `send_async`, but I don't think we can easily achieve this in a good way, and while the wallet is syncing we should not really trigger actions anyway. So I think this is fine :)

Reply:
Just tried it: it's not blocking, you can go out of the wallet. I was able to open a position but I think the sync was already done by that time :)